roles/tftp-server/tasks/main.yml

```diff
@@ -11,10 +18,27 @@
     remote_src: true
     creates: /srv/tftp/pxelinux.0
 
+- name: create dir for configs
+  file:
+    path: "{{ web_root }}/{{ item }}"
+    state: directory
+    recurse: true
+    # owner:  "{{ user_name }}"
+    # group:  "{{ user_name }}"
+  with_items:
+  - lc
+  - d-i/xenial
+
 - name: write preseed.cfg
   template:
     src: preseed.cfg.j2
-    dest: /srv/tftp/preseed.cfg
+    # dest: /srv/tftp/d-i/xenial/preseed.cfg
+    dest: "{{ web_root }}/d-i/xenial/preseed.cfg"
+
+- name: push late_command.sh
+  copy:
+    src: files/late_command.sh
+    dest: "{{ web_root }}/lc/late_command.sh"
```

Comment:
lc or late_command can have all the things that stem from d-i preseed/late_command

hmm, this too much text, but maybe you will find it useful.

I was just reading some docs that talked about di constructing a url like d-i/$flavor/late_command.sh
if we are going to put much effort into dir names, we should figure that out. 

or if you want to change lc to something, here are the things that should be grouped:

pxe - initial file and all the things chain loaded, menu files, memtest, ipxe, 
preseed - d-i/flavor/preseed.cfg (and I like to break that into general, disk, local, maybe also user.
early_command - I use this for debugging installs: scripts to let me ssh into the installer, and extract logs.   I hear it is where fancy raid setup and stuff can be done that preseed can't.
late_command - late_command.sh, and it's deps? like the blackmagic.deb, shotcut tarball, ssh keys, maybe the ansible files.  

and maybe boot_images - for sure ipxe.usb (for devices that don't pxe boot) and maybe the installer for the dc-vid install from thumb drive way.